src/processing/README.md

```diff
@@ -0,0 +1,47 @@
+# Processing
+
+> Implementation of [OIS processing](https://docs.api3.org/reference/ois/latest/processing.html).
+
+The pre/post processing is only supported for Node.js environments and uses internal Node.js modules.
+
+## Getting started
+
+1. Install `zod` which is a peer dependency of this module. Zod is used for validating the logger configuration.
+
+## Documentation
```

Comment:
@andreogle How do you feel about this documentation? I think it tells you exactly what you need (what are the functions, arguments and some brief docs), but it can also get outdated pretty quickly.

Reply:
I'm not sure - this isn't something I have too much experience with.

I like that everything is explicit, although the types aren't necessarily clear or where to find them. e.g. how do I know what `ApiCallParameters` are or where to find the definition - this is my biggest concern.

I thought maybe we could use JSDoc's website generator (since you've already got the syntax here) but that would mean both JSDoc AND TS. From a quick search, there do seem to be options for TS (https://github.com/TypeStrong/typedoc) but I have no idea about how this would go

Otherwise I guess manual documentation is fine for now? Links to the argument types might be extra nice if possible